pym/portage/manifest.py

```diff
@@ -57,18 +57,27 @@ def manifest2AuxfileFilter(filename):
 def manifest2MiscfileFilter(filename):
 	return not (filename == "Manifest" or filename.endswith(".ebuild"))
 
-def guessManifestFileType(filename):
+def guessManifestFileType(filename, is_pkg = True):
```

Comment:
I really don't like this function. Is it really necessary to classify files based on extensions like this?